src/table/components/__tests__/TableWrapper.spec.jsx

```diff
@@ -111,10 +113,45 @@ describe('<TableWrapper />', () => {
         keyboard={keyboard}
         translator={translator}
         rect={rect}
+        announcer={announcer}
       />
     );
+
     fireEvent.change(getByTestId('select'), { target: { value: 25 } });
+
     expect(setPageInfo).to.have.been.calledWith({ page: 0, rowsPerPage: 25 });
+    expect(announcer).to.have.been.calledOnceWith({
+      keys: [['SNTable.Pagination.RowsPerPageChange', sinon.match.any]],
+      politeness: 'assertive',
+    });
+  });
+
+  it('should call setPageInfo when changing page', () => {
+    rect = {
+      width: 651,
+    };
+    const { getByTestId } = render(
+      <TableWrapper
+        tableData={tableData}
+        pageInfo={pageInfo}
+        setPageInfo={setPageInfo}
+        constraints={constraints}
+        selectionsAPI={selectionsAPI}
+        rootElement={rootElement}
+        keyboard={keyboard}
+        translator={translator}
+        rect={rect}
+        announcer={announcer}
+      />
+    );
+
+    fireEvent.change(getByTestId('pagination-dropdown'));
+
+    expect(setPageInfo).to.have.been.calledOnce;
+    expect(announcer).to.have.been.calledOnceWith({
+      keys: [['SNTable.Pagination.PageStatusReport', sinon.match.any]],
```

Comment:
Change `sinon.match.any` to the actual value of new page

Reply:
✅ Fixed